crates/biome_css_analyze/tests/specs/a11y/useGenericFontNames/valid.css

```
@@ -9,6 +9,8 @@ b { font-family: initial; }
 b { font-family: unset; }
 b { font-family: serif; }
 b { font-family: sans-serif; }
+b { font-family: math; }
+b { font-family: STIX Two Math, math; }
```

Stylelint (17.14.0)

[error] Expected quotes around "STIX Two Math" (font-family-name-quotes)

(font-family-name-quotes)